### PR TITLE
nodejs-ko: Update Package Manager Doc

### PR DESCRIPTION
**Pull request by uyu423**
Updated the Korean content of the package manager documentation based on b9a8c09 commit.

### CODE REVIEW

**Review by marocchino — APPROVED** [review]

File: locale/ko/download/package-manager.md [hunk 1]
```diff
@@ -245,51 +245,39 @@ Run as root on RHEL, CentOS or Fedora, for Node.js v6 LTS:
 CentOS 5)의 Node.js 패키지도 사용할 수 있습니다. 설치되어 있지 않다면 설정 스크립트가 확인하고
 안내할 것입니다.
 
-Node.js v6 LTS를 설치하려면 RHEL, CentOS, Fedora에서 root로 실행하세요.
+RHEL, CentOS, Fedora에서 Node.js v8을 사용하려면 다음을 실행합니다.
```
Comment (marocchino) [inline locale/ko/download/package-manager.md]: LTS is ommited here.

File: locale/ko/download/package-manager.md [hunk 2]
```diff
@@ -458,28 +448,39 @@ Additionally, versions of **CentOS** and **Scientific Linux** corresponding to t
 * **RHEL 6** (i686/x86_64)
 * **RHEL 7** (x86_64)
 
+RHEL 6에서는 더이상 EPEL로 지원되지 않습니다.하지만 [Red Hat Software Collections](https://www.softwarecollections.org/en/scls/?search=nodejs)에서 사용 할 수 있습니다.
```
Comment (marocchino) [inline locale/ko/download/package-manager.md]: (maybe)

EPEL으로 하던 RHEL 6지원은 더이상 없습니다. 하지만 ~~~를 사용하실 수 있습니다.

File: locale/ko/download/package-manager.md [hunk 3]
```diff
@@ -575,7 +584,7 @@ pkgsrc에서 Node.js를 설치할 수 있습니다
 cd /usr/pkgsrc/lang/nodejs && make install
 ```
 
-pkgin로 바이너리 패키지를 설치하는 방법도 있습니다.(사용하는 플랫폼에서 가능하다면)
+사용하는 플랫폼에서 가능하다면 pkgin로 바이너리 패키지를 설치하는 방법도 있습니다.
```
Comment (marocchino) [inline locale/ko/download/package-manager.md:586]: 👍

File: locale/ko/download/package-manager.md [hunk 4]
```diff
@@ -245,51 +245,39 @@ Run as root on RHEL, CentOS or Fedora, for Node.js v6 LTS:
...
-Node.js 0.10을 사용한다면 다음을 실행하세요.
+Node.js 9를 사용한다면 다음을 실행하세요.
 
 ```bash
-curl --silent --location https://rpm.nodesource.com/setup | bash -
+curl --silent --location https://rpm.nodesource.com/setup_9.x | sudo bash -
 ```
 
-그 다음 root 계정으로 설치하세요.
+그 다음 아래의 명령어로 설치하세요.
```
Comment (marocchino) [inline locale/ko/download/package-manager.md:277]: (maybe) you can ommit "아래의 명령어로"

File: locale/ko/download/package-manager.md [hunk 5]
```diff
@@ -135,10 +135,10 @@ sudo apt-get install -y build-essential
 * **Debian testing** (stretch, aliased to jessie)
 * **Debian unstable** (sid)
 -->
-Node.js v7을 사용하고 싶다면 다음을 실행합니다.
+Node.js 9을 사용하고 싶다면 다음을 실행합니다.
```
Comment (yous) [inline locale/ko/download/package-manager.md]: 9을 → 9를

File: locale/ko/download/package-manager.md [hunk 6]
```diff
@@ -458,28 +448,39 @@ Additionally, versions of **CentOS** and **Scientific Linux** corresponding to t
...
 
-* **Fedora Rawhide** (i686/x86_64/armv7hl)
-* **Fedora 21** (i686/x86_64/armv7hl)
-* **Fedora 20 (Heisenbug)** (i686/x86_64/armv6hl/armv7hl)
-* **Fedora 19 (Schrödinger's Cat)** (i686/x86_64/armv7hl)
+* **Fedora Rawhide** (i686/x86_64/armv7hl/aarch64/ppc64/ppc64le/s390x)
+* **Fedora 26** (i686/x86_64/armv7hl/aarch64/ppc64/ppc64le)
+* **Fedora 25** (i686/x86_64/armv7hl)
+* **Fedora 24** (i686/x86_64/armv7hl)
+(Schrödinger's Cat)** (i686/x86_64/armv7hl)
```
Comment (yous) [inline locale/ko/download/package-manager.md]: This line should be deleted.

File: locale/ko/download/package-manager.md [hunk 7]
```diff
@@ -507,11 +508,19 @@ pkg install node-devel
 
 Node.js는 ports 시스템으로 설치할 수 있습니다.
 
+**FreeBSD**
```
Comment (yous) [inline locale/ko/download/package-manager.md]: `:` is omitted at the end.

File: locale/ko/download/package-manager.md [hunk 8]
```diff
@@ -507,11 +508,19 @@ pkg install node-devel
...
+
 ```bash
 /usr/ports/www/node
 ```
 
-개발 버전도 ports에서 사용할 수 있습니다.
+**OpenBSD**
```
Comment (yous) [inline locale/ko/download/package-manager.md]: `:` is omitted at the end.

**Comment by uyu423** [comment]
@marocchino  @yous Thank you for your review. I will additional PR after the document modify.

**Comment by uyu423** [comment]
@marocchino @yous Document content modified by your comments.

File: locale/ko/download/package-manager.md [hunk 9]
```diff
@@ -458,28 +448,38 @@ Additionally, versions of **CentOS** and **Scientific Linux** corresponding to t
 * **RHEL 6** (i686/x86_64)
 * **RHEL 7** (x86_64)
 
+RHEL 6에서는 더이상 EPEL로 지원되지 않습니다.하지만 [Red Hat Software Collections](https://www.softwarecollections.org/en/scls/?search=nodejs)를 사용 할 수 있습니다.
```
Comment (yous) [inline locale/ko/download/package-manager.md]: 않습니다.하지만 → 않습니다. 하지만

**Comment by uyu423** [comment]
@yous  fix typo complete

**Review by yous — APPROVED** [review]

**Pull request merged**